### PR TITLE
CATTY-497 Replace TTTAtributedLabel with ActiveLabel

### PR DESCRIPTION
**Pull request by amelak9**
This dependency should be replaced with the following: https://github.com/optonaut/ActiveLabel.swift
Only URLs should be clickable. If clicked, the URL should open in Safari (similar to attributedLabel:didSelectLinkWithURL:).
Links should be displayed in our default iceblue color, no underline.
Actions for phone numbers are not needed, do not forget to remove attributedLabel:didSelectLinkWithPhoneNumber.

Add your new code to the newly created Swift extension for the ProjectDetailStoreViewController.
- [x] Include the name of the Jira ticket in the PR’s title
- [x] Verify that the Jira ticket is in the status *Ready for Development*
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#catty* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: src/Catty.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -11390,15 +11388,14 @@
...
 			inputPaths = (
 				"$(SRCROOT)/Carthage/Build/iOS/Bohr.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/TTTAttributedLabel.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/ZipArchive.framework",
+				"$(SRCROOT)/Carthage/Build/iOS/ActiveLabel.framework",
 			);
 			name = "Carthage Copy Frameworks Run Script";
 			outputPaths = (
 				"$(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/Bohr.framework",
-				"$(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/M13ProgressSuite.framework",
```
Comment (aeluna) [inline src/Catty.xcodeproj/project.pbxproj:11399]: Why did you remove this line? Is it not needed anymore?

File: src/Catty/ViewController/Download/Details/ProjectDetailStoreViewController.m [hunk 2]
```diff
@@ -160,21 +160,13 @@ -(void)downloadAgain:(id)sender
     [self downloadWithName:duplicateName];
 }
 
-#pragma mark - TTTAttributedLabelDelegate
-- (void)attributedLabel:(TTTAttributedLabel *)label didSelectLinkWithURL:(NSURL *)url
+- (void)reloadWithProject:(CatrobatProject *)loadedProject
```
Comment (aeluna) [inline src/Catty/ViewController/Download/Details/ProjectDetailStoreViewController.m:163]: Could it be, that this method was added by mistake, or do you really need it in your PR?
Since the method was removed in the develop branch a few days ago.

File: src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift [hunk 3]
```diff
@@ -406,8 +404,11 @@ let kHTMLAHrefTagPattern = "href=\"(.*?)\""
         var mutableActiveLinkAttributes: [AnyHashable: Any] = [:]
         mutableActiveLinkAttributes[kCTForegroundColorAttributeName as String] = UIColor.brown
         mutableActiveLinkAttributes[kCTUnderlineStyleAttributeName as String] = NSNumber(value: false)
+    }
 
-        label.linkAttributes = mutableLinkAttributes
-        label.activeLinkAttributes = mutableActiveLinkAttributes
+    private func attributedLabel(_ label: ActiveLabel?, didSelectLinkWith url: URL?) {
```
Comment (aeluna) [inline src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift:409]: This function doesn't seem to do, what it is supposed to do.
Since you now use a ActiveLabel, you also need to handle it.
I would suggest having a look at the API calls for ActiveLabel. Especially at the *handleURLTap* :)

File: src/Catty.xcodeproj/project.pbxproj (hunk 1 above)
Comment (amelak9) [reply src/Catty.xcodeproj/project.pbxproj:11399]: This is the framework that I removed few months ago

**Review by aeluna — APPROVED** [review]
Looks great!
